.github/workflows/swift-analysis.yml

```
@@ -0,0 +1,46 @@
+      - name: Build project
+        run: |
+          xcodebuild -scheme YourAppScheme -destination 'platform=iOS Simulator,name=iPhone 15' build-for-testing
+        continue-on-error: true
+
+      - name: Run Tests
+        run: |
+          xcodebuild test -scheme YourAppScheme -destination 'platform=iOS Simulator,name=iPhone 15'
+        continue-on-error: true
+
+      - name: Setup Python
+        uses: actions/setup-python@v4
```

actionlint (1.7.7)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)